packages/manager/apps/web/client/app/hosting/website-coach/website-coach.html

```diff
@@ -0,0 +1,63 @@
+<div class="container-fluid px-5 mt-5">
+    <div class="row">
+        <div class="col-md-4">
+
+            <!-- Screenshot visualizer -->
+            <div class="website-coach-visualizer">
+                <img class="website-coach-visualizer__computer"
+                    src="/assets/images/website-coach-computer.png"
+                    alt="{{$ctrl.attachedDomain}}">
```

Comment:
What's the current gain to use `ng-attr-alt` over `alt` here (performance) ?
The declaration is the same for both `ng-attr-alt="{{$ctrl.youpi}}"`/ `alt="{{$ctrl.youpi}}"`.
I use that mainly when I don't want the attribute to be set if undefined.

Reply:
I suppose it's like `href` and `ng-href`, it is not valid from an HTML point of view

Reply:
Ok, didn't thought about that. I'll update that if all of you prefer.
For me, it was already compiled before the validation. Because it has no purpose to validate something with empty data.

Reply:
With that change, `htmlhint` is not really happy (alt-require)

Reply:
```suggestion
                    alt="" data-ng-attr-alt="{{$ctrl.attachedDomain}}>
```